### DISCUSSION TITLE
Is server side analytics possible ?

### DISCUSSION THREAD

**Discussion by michael-picard** [plausible/analytics#1260] [Anything Else]
Hello there, 

I would like to track api requests to our server in a dedicated Plausible site, I was thinking about sending a post request to https://plausible.io/api/event from our server each time one of our api endpoint is hit.

Is this possible and is there anything I should know before I begin ?

**Comment by metmarkosaric** [comment]
we've now documented how to do this here https://plausible.io/docs/events-api

**Reply by michael-picard** [reply]
Awesome thanks !
